examples/mood-classifiers/src/featureExtraction.js

```diff
@@ -0,0 +1,45 @@
+importScripts('./lib/essentia.js-model.js');
+importScripts('./lib/essentia-wasm.module.js');
+
+const essentiaWASM = Module; // Module is EssentiaWASM, tweaked ES module version without exports
```

Comment:
The latest builds have `.umd.js` builds which can be imported using `importScripts` without this manual tweak. You can use `npm run build-js-api` for the latest builds from the branch.